cypress/support/fragments/courses/coursesView.js

```diff
@@ -0,0 +1,53 @@
+import { HTML, including } from '@interactors/html';
+import { Button, Section, MultiColumnListCell, MultiColumnListRow } from '../../../../interactors';
+import Courses from './courses';
+import NoteEditForm from '../notes/existingNoteEdit';
+
+const notesSection = Section({ id: 'viewCourseNotes' });
+
+export default {
+  close() {
+    cy.do(Button({ icon: 'times' }).click());
+    Courses.waitLoading();
+  },
+
+  waitLoading() {
+    cy.expect([notesSection.exists(), Button('Actions').exists()]);
+  },
+
+  checkNotesSectionContent(notes = []) {
+    // wait for section to load
+    cy.wait(900);
```

Comment:
What if we replace this with `this.waitLoading();`?
doesn't it gonna work? :)

Reply:
fixed :)